nnstreamer_example/example_filter_performance_profile/nnstreamer_example_filter_performance_profile.c

```diff
@@ -414,11 +412,6 @@ _set_and_parse_option_info (int argc, char *argv[], nnstrmr_app_context_t * ctx)
   }
   ctx->input_src_height = height;
 
-  if (width == -1) {
-    width = DEFAULT_WIDTH_INPUT_SRC;
-  }
-  ctx->input_src_width = width;
```

Comment:
```ctx->input_src_width = width;```
Do you really need to remove this line, too?

Reply:
Those lines from 417-420 are just a duplication of the lines (407-410). The line 420 is the same line of 410.

Reply:
ah. I've got it. LGTM